src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_process_net_command_json.py

```diff
@@ -280,6 +280,13 @@ def _set_debug_options(self, py_db, args):
         if bool(path_mappings):
             pydevd_file_utils.setup_client_server_paths(path_mappings)
 
+        if self._debug_options.get('REDIRECT_OUTPUT', False):
```

Comment:
I think it'd be nice to have unit-tests with and without output redirect in `ptvsd`. I searched for `redirect` in the ptvsd tests and only found one reference in the `DebugSession` constructor (so, it seems we only have tests which always redirect on `ptvsd`).